template.html

```diff
@@ -29,5 +29,6 @@
     </div>
 
     <a href="x-_________.html">Restart</a>
+    <script src="template.js"></script>
```

Comment:
Wieso nicht defer und in den head?

Reply:
![image](https://github.com/user-attachments/assets/c52487b1-dac0-4459-a520-c08d1346b1f3)